risc0/zkvm/src/host/protos/api.proto

```diff
@@ -172,6 +172,7 @@ message ProverOpts {
   bool prove_guest_errors = 2;
   ReceiptKind receipt_kind = 3;
   repeated base.Digest control_ids = 4;
+  uint64 segment_limit_po2 = 5;
```

Comment:
```suggestion
  uint64 max_po2 = 5;
```

Reply:
Addressed in https://github.com/risc0/risc0/commit/cc0c9dce0fe90ab8ed20bdd03583b26757ea7ac1